api/valor_api/backend/metrics/classification.py

```diff
@@ -82,10 +83,29 @@ def _compute_curves(
                 models.Label,
                 models.Label.id == predictions.c.label_id,
             )
+            .join(
+                models.Datum,
+                models.Datum.id == models.Annotation.datum_id,
+            )
             .where(predictions.c.score >= threshold)
             .alias()
         )
 
+        groundtruths_with_uids = (
```

Comment:
could we add the changes here to the original `groundtruths` instead of creating a new subquery?